### PR TITLE
Use dind image

### LINKED ISSUE #57: Use official docker in docker image
Any reason we aren't using https://hub.docker.com/_/docker/ instead of the rancher base?

### PR DESCRIPTION
**Pull request by donny-dont**
Tested on a 0.5 installation. Should be good to go.

Fixes #57

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,8 +2,8 @@
...
 
-FROM rancher/docker:v1.10.2
+FROM docker:1.11-dind
 
-ADD drone-docker /usr/bin/
-VOLUME /var/lib/docker
-ENTRYPOINT ["/usr/bin/dockerlaunch", "/usr/bin/drone-docker"]
+ADD drone-docker /go/bin/
+
+ENTRYPOINT ["/go/bin/drone-docker"]
```
Comment (bradrydzewski) [inline Dockerfile]: We need to reserve the ability to mount /go so let's add to /bin

Comment (bradrydzewski) [inline Dockerfile]: Or /usr/bin

**Comment by donny-dont** [comment]
Updated @bradrydzewski

**Comment by donny-dont** [comment]
Matches the git plugin now

**Comment by benschumacher** [comment]
LGTM

**Comment by bradrydzewski** [comment]
LGTM

**Pull request merged**